recipes/rosbuild/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: ros-rosbuild
+  version: 1.14.6
+
+source:
+  url: https://github.com/ros-gbp/ros-release/archive/release/melodic/rosbuild/1.14.6-0.tar.gz
+  fn: ros-rosbuild-1.14.6.tar.gz
+  sha256: 3c95011a3c521dc30e8b7f4399fdc46f0e3702978b9060f309529e6e8f396e43
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - ros-conda-mutex * melodic
```

Comment:
Let me understand this part
are you getting any version with the build string ``melodic`` ?
I am a bit puzzled by that xD

Reply:
ROS has a notion of named distributions for the 5k+ packages, we only package `melodic` for now, but we will enable other values for the build string later on.